src/worker.py

```
@@ -6314,6 +6371,27 @@ async def on_fetch(request, env):
         return await render_500(request, env)
 
 
+def _get_study_groups_module():
+    cached = sys.modules.get("study_groups") or sys.modules.get("src.study_groups")
+    if cached is not None:
+        return cached
+
+    try:
+        mod = importlib.import_module("study_groups")
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)